news/2021-06-30-featured-artist-track-updates-summer-2021.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: post
+title: "Featured Artist Track Updates: Summer 2021"
+date: 2021-06-30 11:30:00 +0000
+---
+
+While we've been busy pumping out new Featured Artists left and right, some of our older talent have been working hard too! We've added over **60** songs from **13** of our current Featured Artists, so come see what's new!
+
+![](/wiki/shared/news/banners/featured-artist.jpg)
+
+We're keeping things simple: **click an artist's name or banner to visit their Featured Artist page.** All new songs are marked with a very very very cool *NEW* icon.
+
+![](/wiki/shared/news/2021-01-02-featured-artist-updates-january-2021/new.png)
+
+New songs are also listed below along with any relevant ranked maps and an audio preview or two. These should give you a quick impression of what's in store, but also remember that all song previews are available from each Featured Artist page!
+
+---
+
+## [BilliumMoto / Dictate / Kurokotei](https://osu.ppy.sh/beatmaps/artists/45)
+
+[![](https://assets.ppy.sh/artists/38/header.jpg)](https://osu.ppy.sh/beatmaps/artists/38)
+
+Coming as a surprise to nobody, we've officially added [**TWC Sound Team "Zeroth Powers" - One One**](https://osu.ppy.sh/beatmapsets/1422682) from the TWC world cup grand finals to the Featured Artist listings for [BilliumMoto](https://osu.ppy.sh/users/3862471), [Dictate](https://osu.ppy.sh/beatmaps/artists/21), and [Kurokotei](https://osu.ppy.sh/beatmaps/artists/58)! Maybe we'll have a listing for *Johnson Solid* one day too.
```

Comment:
```suggestion
Coming as a surprise to nobody, we've officially added [**TWC Sound Team "Zeroth Powers" - One One**](https://osu.ppy.sh/beatmapsets/1422682) from the [osu!taiko World Cup 2021](/wiki/Tournaments/TWC/2021) Grand Finals to the Featured Artist listings for [BilliumMoto](https://osu.ppy.sh/users/3862471), [Dictate](https://osu.ppy.sh/beatmaps/artists/21), and [Kurokotei](https://osu.ppy.sh/beatmaps/artists/58)! Maybe we'll have a listing for *Johnson Solid* one day too.
```
"TWC world cup" is an incorrect expression here, as "TWC" already contains the phrase "world cup". Also linking to the respective tourney wiki would be nice, but not mandatory.